pages/project/add-user/index.js

```diff
@@ -0,0 +1,50 @@
+const { stringify } = require('qs');
+const { page } = require('@asl/service/ui');
+const { form } = require('../../common/routers');
+const getSchema = require('./schema');
+
+module.exports = () => {
+  const app = page({ root: __dirname });
+
+  app.use((req, res, next) => {
+    req.model = {
+      id: `${req.project.id}-add-user`
+    };
+    const query = {
+      filters: {
+        roles: ['holc']
+      }
+    };
+    res.locals.static.profilesHolcFiltered = `${req.buildRoute('profile.list')}?${stringify(query)}`;
+    next();
+  });
+
+  app.use(form({
+    configure: (req, res, next) => {
+      req.api(`/establishment/${req.establishmentId}/profiles`, { query: { limit: 'all' } })
+        .then(({ json: { data } }) => {
```

Comment:
is data always an array?

Reply:
If the API has returned a 200 then yes, it should be. 

At least, if it's not then it's ok for things to loudly blow up I think.

Reply:
you'll always get at least yourself :)